diff -rupN mpc-0.35.old/src/options.c mpc-0.35/src/options.c
--- mpc-0.35.old/src/options.c	2023-12-21 11:37:29.000000000 +0100
+++ mpc-0.35/src/options.c	2025-08-06 12:19:21.780390858 +0200
@@ -40,7 +40,7 @@ struct Options options = {
 static const struct OptionDef option_table[] = {
 	{ 'v', "verbose", NULL, "Give verbose output" },
 	{ 'q', "quiet", NULL, "Suppress status message" },
-	{ 0, "no-status", NULL, "synonym for --quiet" },
+	{ 'q', "no-status", NULL, "synonym for --quiet" },
 	{ 'h', "host", "<host>", "Connect to server on <host>" },
 	{ 'P', "password", "<password>", "Connect to server using password <password>" },
 	{ 'p', "port", "<port>", "Connect to server port <port>" },
