diff -rupN mpc-0.35.old/doc/index.rst mpc-0.35/doc/index.rst
--- mpc-0.35.old/doc/index.rst	2025-08-06 12:06:07.827969700 +0200
+++ mpc-0.35/doc/index.rst	2025-08-06 12:16:21.079668478 +0200
@@ -455,6 +455,7 @@ Other Commands
    ================== ======================================================
    %totaltime%        The total duration of the song.
    %currenttime%      The time that the client is currently at.
+   %currenttimems%    The time that the client is currently at, in milliseconds.
    %percenttime%      The percentage of time elapsed for the current song.
    %songpos%          The position of the current song within the playlist.
    %length%           The number of songs within the playlist
diff -rupN mpc-0.35.old/src/status_format.c mpc-0.35/src/status_format.c
--- mpc-0.35.old/src/status_format.c	2025-08-06 12:14:11.056042044 +0200
+++ mpc-0.35/src/status_format.c	2025-08-06 12:17:44.220443401 +0200
@@ -54,6 +54,9 @@ status_value(const struct mpd_status *st
 		unsigned elasped = mpd_status_get_elapsed_time(status);
 		snprintf(buffer, sizeof(buffer), "%u:%02u",
 			elasped / 60, elasped % 60);
+	} else if (strcmp(name, "currenttimems") == 0) {
+		unsigned elapsed_ms = mpd_status_get_elapsed_ms(status);
+		snprintf(buffer, sizeof(buffer), "%u", elapsed_ms);
 	} else if (strcmp(name, "percenttime") == 0) {
 		sprintf(buffer, "%3u%c", elapsed_percent(status), '%');
 	} else if (strcmp(name, "state") == 0) {
